include/net/net_core.h

```diff
@@ -43,8 +43,14 @@ extern "C" {
 /** @cond INTERNAL_HIDDEN */
```

Comment:
Typo in commit subject and body, s/thead/thread/

Reply:
Thanks, fixed.